runtime/moonbase/src/precompiles.rs

```diff
@@ -14,8 +14,6 @@
 // You should have received a copy of the GNU General Public License
 // along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
 
-#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
I'm surprised to see this one. We need this create to build to `no_std` right?

Reply:
It is a crate attribute and not a block attribute, as you can see by the fact that it starts with #! instead of #. Crate attributes must be placed only once at the root of the crate, and already act in the whole crate, anywhere else they are useless.